Modify title and Abstract (#52)
dashpay/dips: dip-0008.md

chain. ChainLocks leverages LLMQ Signing Requests/Sessions to accomplish
this.

[The original proposal by Satoshi Nakamoto](https://www.bitcoin.com/bitcoin.pdf) 

> provided a novel solution to the Byzantine general's problem.  In some publications, 

Reviewer: ```suggestion
provided a novel solution to the Byzantine general's problem. In some publications, 
```

> this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provide for 

Reviewer 2: ```suggestion
this new consensuses mechanism is called Nakamoto Consensus.  ChainLocks provide for 
```

> this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provide for 

Reviewer 2: ```suggestion
this new consensuses mechanism is called a Nakamoto Consensus.  ChainLocks provides for 
```
Although "ChainLocks" ends with an 's', it's normally been treated as a singular noun

> this new consensuses mechanism is called a Nakamoto consensus.  ChainLocks provides for 

Reviewer: ```suggestion
this new consensuses mechanism is called a Nakamoto consensus. ChainLocks provides 
```

> an advancement of this consensus mechanism which mitigates a 51% attack.  We call 

Reviewer 2: ```suggestion
an advancement of this consensus mechanism which mitigates 51% attacks.  We call 
```
above sounds better imo

> an advancement of this consensus mechanism which mitigates 51% attacks.  We call 

Reviewer: ```suggestion
an advancement of this consensus mechanism which mitigates 51% attacks. We call 
```

> the updated consensus mechanism that ChainLocks provide a post Nakamoto consensus.  

Reviewer 2: ```suggestion
the updated consensus mechanism that ChainLocks provides a post Nakamoto consensus.  
```

Author: Thank you!
